python/ciqueue/distributed.py

```diff
@@ -0,0 +1,148 @@
+import os
+import time
+import math
+from redis import ConnectionError
+
+from ciqueue.static import Static
+
+
+class LostMaster(StandardError):
+  pass
+
+
+class Base(object):
+  def __init__(self, redis, build_id):
+    self.redis = redis
+    self.build_id = str(build_id)
+    self.is_master = False
+    self._scripts = {}
+
+  def key(self, *args):
+    return ':'.join(['build', self.build_id] + map(str, args))
+
+  def wait_for_master(self, timeout=10):
+    if self.is_master:
+      return True
+
+    for i in range(timeout * 10 + 1):
```

Comment:
If you're not using `i` just put a `_` here. Is the idea to wait for 11*time.sleep(0.1) seconds?